lib/dat-worker-pool.rb

```diff
@@ -125,7 +130,6 @@ def force_shutdown(timeout, backtrace)
       worker.join rescue false
       @workers.delete(worker)
     end
-    raise error if @debug
```

Comment:
@kellyredding - As we discussed, there is really no impact from this change. I can't remember what I ever used it for; my best guess is that it was a way for me to hard signal to myself that it was forced to shut down. It may also have been legacy from the original implementation of dat worker pool that was carried through changes just because it was there before.

I removed it because I didn't see a need for a debug flag anymore since it is no longer tied to the logging and this was the only other place it was being used.